rest/resource-server/src/main/java/org/eclipse/sw360/rest/resourceserver/admin/fossology/Sw360FossologyAdminServices.java

```diff
@@ -126,4 +126,10 @@ private void serveCheckConnection() {
             throw new RuntimeException("Connection to Fossology server Failed.");
         }
     }
+
+    public Map<String, Set<String>> getConfig() throws TException {
```

Comment:
This operation should be possible only for `ADMIN` users. Please add a check for return 403 for other users.

Reply:
Made the changes as requested.